tasks/test.yaml

```
@@ -12,9 +12,22 @@ tasks:
 
   - name: ingress
     actions:
-      - description: Sonarqube UI Health Check
-        wait:
-          network:
-            protocol: https
-            address: sonarqube.uds.dev
-            code: 200
+      - description: SonarQube UI Status Check
+        maxRetries: 30
+        cmd: |
+          STATUS=$(curl -s 'https://sonarqube.uds.dev/api/system/status' | yq '.status')
```

```suggestion
          STATUS=$(curl -s 'https://sonarqube.uds.dev/api/system/status' | ./uds zarf tools yq '.status')
```

change to `./uds zarf tools yq`